.github/workflows/legacy-git.yml

```diff
@@ -0,0 +1,25 @@
+name: legacy-git
+
+on:
+  pull_request:
+
+  push:
+    branches:
+      - master
+      - 3.2
+
+jobs:
+  legacy_git:
+    name: Verify behavior under old git
+    runs-on: ubuntu-16.04
```

Comment:
Should we use `ubuntu-latest` ? Because this action run with container environment. So, It's not affect with host version.

Reply:
@hsbt I think it is good idea to specify system version since `ubuntu-latest` can change at any time to new version which could potentially break CI randomly.

Reply:
I'll use 20.04.

Reply:
👍  I'm afraid of ubuntu 16.04 is EOL at 2021/4. 20.04 is nice to me.

Reply:
I was also thinking of verifying that this container _does_ ship an old git, but what I heard from CentOS is that their strong point is long term support (centos 7 is supported until 2024), so I don't think `git` is going to get any updates here.

Reply:
I guess I could add a sha checksum after the image name to make sure that it's really frozen just in case.

Reply:
Ended up doing that. It shoud be ok now!